doc/guide/build.hpp

```diff
@@ -228,8 +228,7 @@ and submit an issue and the mlpack developers will quickly help you figure it
 
 https://github.com/mlpack/mlpack
 
-Alternately, mlpack help can be found in IRC at \#mlpack on irc.
-node.net.
+Alternately, mlpack help can be found in IRC at \#mlpack on irc.freenode.net.
```

Comment:
Pretty sure `irc.freenode.net` doesn't work anymore, but `chat.freenode.net` does which is also mentioned on the website.

Reply:
Done!